src/System.IO.Compression/ref/Compat/System.IO.Compression.csproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Since this is in a non-traditional folder structure I would expect you would need to set ContractProject in the facade for this in order to get it to pick-up this version of the contract. Same with System.Net.Http.